bool activate
bool go
bool autorelease_open
bool autorelease
uint8 position
uint8 speed
uint8 force